readme.md

```diff
@@ -76,32 +74,34 @@ You have already accomplished the first step by reading this readme. Congratz!
 The second one is only slightly harder by joining us in [Riot.im#inexor](https://riot.im/app/#/room/#inexor:matrix.org), consider the [contact](https://github.com/inexorgame/inexor-core/wiki/Contact) page in our wiki for more ways to chat.
 
 To contribute to the project and merge your changes into the master branch you need to convince enough members that the changes are an improvement.
-If you keep communicating what you are doing, take into account feedback and tips from others as well as giving insights into your decision making process, it won't be an issue.
+If you keep communicating what you are doing, take into account feedback and tips from others as well as giving insights into your decision-making process, it won't be an issue.
```

Comment:
I think we should rephrase this sentence.
`It probably won't be an issue. However you should still communicate what you're doing and of course take feedback and tips from others into account.` or sth like that. or just don't make it look so negatively that we review their PR first ^^

Reply:
Done. (However, I didn't introduce this sentence, just fixed grammar.)